test/pageobjects/LoginPage.js

```diff
@@ -1,7 +1,3 @@
-import Base from '../test/pageobjects/Base.js'
+class LoginPage {}
 
-class LoginPage extends Base {
-    
-}
-
-export default new LoginPage()
\ No newline at end of file
+module.exports = new LoginPage();
```

Comment:
I'd suggest removing this file as it does not contain any methods.

Reply:
Sure.